spec/features/visits/03_users_can_edit_a_visit_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+feature "users can edit visits within a vacation" do
+  scenario "signed in user can edit a visit they created" do
+    visit "/"
+    click_link "Sign in with Google"
+    park = FactoryGirl.create(:park, name: "Kenai Fjords")
+    vacation = FactoryGirl.create(:vacation, user: User.last,
```

Comment:
One of the problems I am having is trying to figure out how to log in as a specific user. I have stubbed out the OmniAuth log in for testing, but I can't figure out how to log in besides using `click_link "Sign in with Google"`